annotations-runtime-schedule/scheduler-quartz-1/src/main/java/com/github/dynamicextensionsalfresco/schedule/quartz/ClusterLockedQuartzJob.java

```diff
@@ -0,0 +1,13 @@
+package com.github.dynamicextensionsalfresco.schedule.quartz;
+
+import org.alfresco.schedule.AbstractScheduledLockedJob;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+
+public class ClusterLockedQuartzJob extends AbstractScheduledLockedJob {
```

Comment:
Identical in the quartz-1 and quartz-2 subproject, can this be moved to the `:annotations-runtime-schedule`?

Reply:
I was intentionally keeping `:annotations-runtime-schedule` free from any quartz or Alfresco dependency.

Reply:
Agreed